glfw/src/Window.zig

```diff
@@ -1422,6 +1424,9 @@ pub inline fn setPosCallback(self: Window, callback: ?fn (window: Window, xpos:
     };
 }
 
+// TODO: don't the calls to `from` in these `set*CallbackWrapper` functions cause a leak?
+// is this what's being reported by valgrind in [#60](https://github.com/hexops/mach/issues/60)?
```

Comment:
accidental?

Reply:
Whoops; I'm not sure how that happened. I'm sure I pulled from main